package.json

```diff
@@ -9,7 +9,21 @@
   "author": "JATB",
```

Comment:
## Add scripts for running `webpack`
- Kindly add the necessary commands (scripts) to run instructions like `build`,  `dev`,  & maybe `start`. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```